src/containers/MintWithTicket/MintWithTicketPage.tsx

```diff
@@ -274,6 +274,13 @@ export function MintWithTicketPageRoot({ token, ticketId, mode }: Props) {
     })
   }, [panelParamsRef, params, setHash, historyContext, withAutoUpdate])
 
+  useEffect(() => {
+    document.documentElement.classList.add(style.black)
```

Comment:
sweet man, maybe just add a comment to give this context in case we ever come across it in the future and wonder why it's there 😄

Reply:
good call ;-)